<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>CreateChannelPage</class>
 <widget class="QDialog" name="CreateChannelPage">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>400</width>
    <height>300</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <widget class="QLabel" name="channelname_lbl">
   <property name="geometry">
    <rect>
     <x>80</x>
     <y>110</y>
     <width>251</width>
     <height>31</height>
    </rect>
   </property>
   <property name="text">
    <string>Channel Name</string>
   </property>
   <property name="scaledContents">
    <bool>false</bool>
   </property>
   <property name="alignment">
    <set>Qt::AlignCenter</set>
   </property>
  </widget>
  <widget class="QLineEdit" name="channelname_led">
   <property name="geometry">
    <rect>
     <x>80</x>
     <y>150</y>
     <width>251</width>
     <height>28</height>
    </rect>
   </property>
  </widget>
  <widget class="QLabel" name="result_lbl">
   <property name="geometry">
    <rect>
     <x>80</x>
     <y>250</y>
     <width>231</width>
     <height>20</height>
    </rect>
   </property>
   <property name="text">
    <string>create Channel result...</string>
   </property>
  </widget>
  <widget class="QWidget" name="">
   <property name="geometry">
    <rect>
     <x>150</x>
     <y>200</y>
     <width>181</width>
     <height>31</height>
    </rect>
   </property>
   <layout class="QHBoxLayout" name="horizontalLayout">
    <item>
     <widget class="QPushButton" name="cancel_pbn">
      <property name="text">
       <string>Cancel</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QPushButton" name="createchannel_pbn">
      <property name="text">
       <string>Create</string>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
